### ISSUE TITLE
iojs 3.1.0 build fails

### ISSUE THREAD

**Issue by jay915** [mapbox/node-sqlite3#498]
Hi,
I'm having trouble using your library with electron (v0.31.0). I have tried to install it using `npm install sqlite3` and it fails with several compile time errors. I'm using iojs 3.1.0.
Please, Give me some solution to use this library with electron (v0.31.0) I'm desperately searching for a database to be used with my electron app and I don't want to ship an application that depends on a standalone database like MongoDB or such.

Thank you in advance

**Comment by hmalphettes** [comment]
@jay915 the issue is that iojs-3.x is not supported: https://github.com/mapbox/node-sqlite3/issues/487
Once https://github.com/mapbox/node-sqlite3/pull/495 is merged it will be all good.

As a temporary solution and to confirm we are not missing something could you try @kkoopa's branch and let us know how it goes?

```
"dependencies": {
  "sqlite3": "git://github.com/kkoopa/node-sqlite3.git#nan2",
}
```

Thanks!

**Comment by jay915** [comment]
The @kkoopa's branch is working for me. Thank you very much. You've really help me a lot.

**Comment by jay915** [comment]
@kkoopa I really apprecitae the help but I've got a new error. Actually when I told you that it's working I tested it with the iojs not electron and now when I try to use it with electron it gives me the following error:

_A dynamic link library (DLL) initialization routine failed._

Please, Help!

**Comment by springmeyer** [comment]
This should be fixed by the v3.1.0 release just published.

**Comment by erikdung** [comment]
I got this error on win32-ia32, please help.

**Comment by mishac** [comment]
I've got the same issue on windows x64:

![image](https://cloud.githubusercontent.com/assets/68727/13954854/e8c5f002-f018-11e5-8d93-ebe96c2fd79a.png)

**Comment by springmeyer** [comment]
@mishac - please do not comment on old/closed issues. Please open a new issue with more details like your node version, how you installed node, and what version of node-sqlite3 you are installing.
